lib/ruby-debug-ide/attach/native_debugger.rb

```diff
@@ -38,8 +40,8 @@ def find_attach_lib(debase_path)
     raise 'Could not find attach library'
   end
 
-  def attach_to_process(pid)
-    execute "attach #{pid}"
+  def attach_to_process
+    execute "attach #{@pid}"
```

Comment:
Still I believe `pid` must be specified in `executable` when running gdb/lldb. AFAIR you've said yourself it does not work otherwise?